macros/src/main/scala/me/rexim/morganey/meta/package.scala

```diff
@@ -0,0 +1,19 @@
+package me.rexim.morganey
+
+import scala.language.experimental.macros
+
+package object meta extends AnyRef
+  with DefaultLiftableInstances
+  with DefaultUnliftableInstances {
+
+  private[meta] trait Quasiquotation {
+    def apply[T](args: T*): Any = macro QuotationMacro.quote
+    def unapply(arg: Any): Any = macro QuotationMacro.unquote
+  }
+
+  implicit class Quasiquote(ctx: StringContext) {
+    object m extends Quasiquotation
+    object lc extends Quasiquotation
```

Comment:
Other names i've thought of: `lambda`, `ast` and `tree`. I didn't like all of those ideas, because names were either too long for my taste (for interpolators) or they were not related to the project. 

Ultimately @rexim has to decide, which name he wants.